server/routes/user.js

```diff
@@ -21,7 +21,22 @@ router.route('/:userId')
   /** DELETE /api/v1/user/:userId - Delete user */
   .delete(validate(paramValidation.byId), userCtrl.deleteUser);
 
+router.route('/:userId/device')
+  /** POST /api/v1/user/:userId/device - Add device to user */
+  .post(validate(paramValidation.addDevice), userCtrl.addDevice);
+
+router.route('/:userId/device/:deviceId')
```

Comment:
I don't really like that approach for routing. Even though there is a some repetition with the current setup, I think it's easier to read as a result.